test/export/test_torchbind.py

```diff
@@ -392,6 +403,52 @@ def forward(self, arg0_1, attr, arg1_1):
     return (getitem_3, add_1)""",  # noqa: B950
         )
 
+    def test_make_fx_tensor_queue_methods(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(3, 2)
+
+            def forward(self, tq, x):
+                tq.push(x.cos())
+                tq.push(x.sin())
+                x_cos = tq.pop() + tq.size()
+                x_sin = tq.pop() - tq.size()
+                return x_sin, x_cos, tq
+
```

Comment:
@ydwu4 we should check that the methods on FakeTensorQueue actually get called. We can do this by having the FakeTensorQueue methods increment a counter when they get called.

Reply:
That's a good idea! I've had it printed 'push'/'pop' and removed them later. I can add the counter.